resources/logging/profile-evaluation.yaml

```diff
@@ -1,8 +1,8 @@
 loki:
   resources:
     limits:
-      cpu: 100m
```

Comment:
The logging component got already removed from main branch

Reply:
removed